keps/sig-network/4004-deprecate-kube-proxy-version/README.md

```diff
@@ -204,11 +204,23 @@ N/A
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-N/A
+  I've tried to simulate this manually by running (using `local-cluster-up.sh`with `PRESERVE_ETCD=true`):
+
```

Comment:
Just to clarify - this was the same node that the field was updated?
I want to ensure that we're not talking about different nodes across runs...

Reply:
I added the steps for each step and showed the node’s uuid to prove that we are always checking the fields of the same node object.

Reply:
This is great - thanks!